src/ExplainProvenance.h

```diff
@@ -185,12 +185,7 @@ class ExplainProvenance {
             return "Relation " + relName + " not found\n";
         }
 
-        // create symbol mask
-        std::vector<RamTypeAttribute> symbolMask(rel->getArity());
-
-        for (size_t i = 0; i < rel->getArity(); i++) {
-            symbolMask.at(i) = RamPrimitiveFromChar(*(rel->getAttrType(i)));
-        }
+        // TODO (darth_tytus): Create a JSON typesystem here?
```

Comment:
Is this resolved? Could you check that the query/explain system (part of provenance) is still working? (NB: we have tests for those). 

If so, please remove the TODO.

Reply:
It seems this method is not used by anything.
Should I remove it, remove TODO or update it?